app/views/wallet/GiftingView/BuildGiftPanel/BuildGiftForm.tsx

```diff
@@ -285,10 +298,11 @@ const BuildGiftForm: FC = () => {
         try {
           setIsAwaitingConformation(true);
 
+          const adjustedValue = Number(values.mobValue) + 0.01;
```

Comment:
Shouldn't this 0.01 (fee) be a constant somewhere?

Reply:
It should be, yes, but anything Fee related is scattered all over the project and I wasn't sure where we wanted constants to live. Is this okay for now until we get to the refactor?

Reply:
Sorry about the mess!